documentation/modules/mtv-performance-recommendation.adoc

```diff
@@ -174,4 +170,43 @@ Testing showed that by breaking one single large plan into multiple moderately s
+
+The following are recommendations that are specific to warm migrations:
+
+[id="mtv-warm-migration-recommendations-disks-in-parallel_{context}"]
+=== Migrate up to 400 disks in parallel
+
+Testing involved migrating 200 VMs in parallel, with 2 disks each using 8 ESX hosts, for a total of 400 disks. No tests were run on migration plans migrating over 400 disks in parallel, so it is not recommended to migrate over this number of disks in parallel.
+
+[id="mtv-warm-migration-recommendations-effects-of-precopy-speed_{context}"]
+=== Migrate up to 200 disks in parallel for the fastest rate
+
+Testing was successfully performed on parallel disk migrations with 200, 300, and 400 disks. There was a decrease in the precopy migration rate, approximately 25%, between the tests migrating 200 disks and those migrating 300 and 400 disks.
+
+Therefore, it is recommended to perform parallel disk migrations in groups of 200 or fewer, instead of 300-400 disks, unless a drop of 25% in precopy speed does not affect your cutover planning.
+
+[id="mtv-warm-migration-recommendations-set-cutover-time_{context}"]
+=== When possible, set cutover time to be immediately after a migration plan starts
+
+To reduce the overall time of warm migrations, it is recommended to set the cutover to occur immediately after the migration plan is started. This will cause the precopy phase of all VMs to run in parallel, subject to the setting of `MAX_VM_INFLIGHT`. This recommendation is valid no matter how many VMs are in the migration plan.
+
+[id="mtv-warm-migration-recommendations-increase-precopy-intevals_{context}"]
+=== Increase precopy intervals between snapshots
+
+If you are creating many migration plans with a single VM and have enough time between the migration start and the cutover, increase the value of the `controller_precopy_interval` parameter to between 120 and 240 minutes, inclusive. The longer setting will reduce the total number of snapshots and disk transfers per VM before the cutover.
+
+[id="mtv-performance-max-values-warm_{context}"]
+== Maximum values tested for warm migrations
+* Maximum number of ESX hosts tested: 8
+* Maximum number of worker nodes: 12
+* Maximum number of VMs in a single migration plan: 200
+* Maximum number of total parallel disk transfers: 400, with 200 VMs, 6 ESXs, and a transfer rate of 667 MB/s
+* Maximum number of number of disks on a single VM migrated: 3
+* Maximum number of parallel disk transfers per ESX host: 68
+* Maximum transfer rate observed of a single disk with no concurrent migrations: 76.5 MB/s
+* Maximum transfer rate observed of multiple disks from a single ESX host: 253 MB/s (concurrent migration of 10 VMs, 1 disk each, 35/50 GiB used per disk)
+* Total transfer rate observed of multiple disks (210) from 8 ESX hosts: 802 MB/s (concurrent migration of 70 VMs, 3 disks each, 35/50 GiB used per disk)
+
+For additional details on performance, see xref:mtv-performance-addendum_mtv[{project-short} performance addendum].
```

Comment:
Delete this line, In the addendum there is only cold migration results

Reply:
@dgur Do you want me to delete it or should I return it to the end of "Maximum values tested for cold migrations" ?